docs/exceptions.md

```diff
@@ -230,6 +288,12 @@ code, avoid `forkIO` or `forkProcess` in favor of the `async` package, as it
 rethrows exceptions from the child threads. (Do not use the function `async`
 itself when you can use `withAsync`, `race`, or `concurrently`).
 
+*Comment:* and when `async` is the only option, be aware of `link` and use it
+if appropriate.
+
+*Comment:* beware useing `withAsync` recursively. It essentially does a bracket,
```

Comment:
Doesn't have to do anything with the exceptions/errors guideline.

Reply:
Sure it does. It's a caveat for the advice about `withAsync`. It can't be used in some cases, due to its properties with respect to exception handling.